Description: honour CFLAGS provided by the environment
 This allows debian/rules to override the CFLAGS variable.
Author: Guus Sliepen <guus@debian.org>

--- wireless-tools-30~pre9.orig/Makefile
+++ wireless-tools-30~pre9/Makefile
@@ -106,7 +106,8 @@ endif
 # Other flags
 #CFLAGS=-Os -W -Wall -Wstrict-prototypes -Wmissing-prototypes -Wshadow \
 	-Wpointer-arith -Wcast-qual -Winline -I.
-CFLAGS=-O2 -W -Wall -Wstrict-prototypes -I.
+CFLAGS?=-O2 -W -Wall -Wstrict-prototypes -I.
+CFLAGS+=-I.
 DEPFLAGS=-MMD
 XCFLAGS=$(CFLAGS) $(DEPFLAGS) $(WARN) $(HEADERS) $(WELIB_FLAG) $(WEDEF_FLAG)
 PICFLAG=-fPIC
